salt/modules/junos.py

```diff
@@ -476,6 +524,13 @@ def rollback(id=0, **kwargs):
     else:
         ret['message'] = 'Rollback succesfull but pre-commit check failed.'
         ret['out'] = False
+
+    if op['unlock']:
+        try:
+            conn.cu.unlock()
+        except:
+            pass
```

Comment:
This is pep20 violation which even I am not in favour of
https://www.python.org/dev/peps/pep-0020/

"Errors should never pass silently."